dpnp/backend/extensions/lapack/gesv.cpp

```diff
@@ -42,49 +43,150 @@ namespace mkl_lapack = oneapi::mkl::lapack;
 namespace py = pybind11;
 namespace type_utils = dpctl::tensor::type_utils;
 
-typedef sycl::event (*gesv_impl_fn_ptr_t)(sycl::queue,
+void common_gesv_checks(sycl::queue &exec_q,
```

Comment:
Can we move that to `gesv_common.hpp` or `gesv_common_utils.hpp` and make it inline rather than using extern specification.

Reply:
Done!